propagator/google_cloud_trace_context/test/opentelemetry/propagator/ottrace/text_map_propagator_test.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Propagator::GoogleCloudTraceContext::TextMapPropagator do
+  class FakeGetter
+    def get(carrier, key)
+      '7ffe3d75a2b8ef468ab34365ee891f08/2141054039718851918;o=0'
+    end
+
+    def keys(carrier)
+      []
+    end
+  end
+
+  class FakeSetter
+    def set(carrier, key, value)
+      carrier[key] = "#{key} = #{value}"
+    end
+  end
+
+  let(:span_id) do
+    'e457b5a2e4d86bd1'
+  end
+
+  let(:trace_id) do
+    '80f198ee56343ba864fe8b2a57d3eff7'
+  end
+
+  let(:trace_flags) do
+    OpenTelemetry::Trace::TraceFlags::DEFAULT
+  end
+
+  let(:context) do
+    OpenTelemetry::Trace.context_with_span(
+      OpenTelemetry::Trace.non_recording_span(
+        OpenTelemetry::Trace::SpanContext.new(
+          trace_id: Array(trace_id).pack('H*'),
+          span_id: Array(span_id).pack('H*'),
+          trace_flags: trace_flags
+        )
+      )
+    )
+  end
+
+  let(:propagator) do
+    OpenTelemetry::Propagator::GoogleCloudTraceContext::TextMapPropagator.new
+  end
+
+  let(:parent_context) do
+    OpenTelemetry::Context.empty
+  end
+
+  let(:gcp_header) do
+    '80f198ee56343ba864fe8b2a57d3eff7/16453819474850114513;o=1'
```

Comment:
Maybe embed `#{trace_id}` and `#{span_id.to_i(16)}` here instead of repeating the values?